orchagent/policerorch.cpp

```diff
@@ -127,186 +389,204 @@ void PolicerOrch::doTask(Consumer &consumer)
 
         auto key = kfvKey(tuple);
         auto op = kfvOp(tuple);
+        auto table_name = consumer.getTableName();
 
-        if (op == SET_COMMAND)
+        if (table_name == CFG_PORT_STORM_CONTROL_TABLE_NAME)
         {
-            // Mark the operation as an 'update', if the policer exists.
-            bool update = m_syncdPolicers.find(key) != m_syncdPolicers.end();
-
-            vector<sai_attribute_t> attrs;
-            bool meter_type = false, mode = false;
-
-            for (auto i = kfvFieldsValues(tuple).begin();
-                    i != kfvFieldsValues(tuple).end(); ++i)
+            storm_status = handlePortStormControlTable(tuple);
+            if ((storm_status == task_success) ||
+                    (storm_status == task_failed))
             {
-                auto field = to_upper(fvField(*i));
-                auto value = to_upper(fvValue(*i));
-
-                SWSS_LOG_DEBUG("attribute: %s value: %s", field.c_str(), value.c_str());
+                it = consumer.m_toSync.erase(it);
+            }
+            else
+            {
+                it++;
+                continue;
```

Comment:
You can put continue outside of this if check. Remove the else part below and retain the code as-is.

Reply:
Done. Please review.